### ISSUE TITLE
Support .vue single file components

### ISSUE THREAD

**Issue by agritheory** [stefanhoelzl/vue.py#2]
[The spec](https://vue-loader.vuejs.org/spec.html) for reference.

```
import Vue from 'vue'
import Router from 'vue-router'
import Hello from '@/components/Hello'
```
I think an implementation of this _exact_ would involve monkey patching the import statement to load .vue files. That seems like maybe not the best idea, but an sfg loader utility/class would be required underneath regardless.

**Comment by stefanhoelzl** [comment]
what is your use case?

for know the concept is to have a pure python project and do not mix .py and .js/.vue files.

you can have a look at [MQTT Dashboard](https://github.com/stefanhoelzl/mqtt-dashboard) to see how to include an external javascript vue library.
see this files [app.py](https://github.com/stefanhoelzl/mqtt-dashboard/blob/master/app/app.py) and [vuepy.yml](https://github.com/stefanhoelzl/mqtt-dashboard/blob/master/app/vuepy.yml)

**Comment by agritheory** [comment]
I want to write python between the `<script>` tags in a `.vue` file. Writing HTML and CSS as docstrings is not as good.

**Comment by stefanhoelzl** [comment]
I agree writing HTML and CSS in python strings is not very comfortable. 

But on the other hand in a .vue file you will miss a lot of handy things for your python code (auto-completion, syntax highlighting, refactoring support, ...)

Since I am depending on brython I have to check if it is possible to hook into the import machinery somehow.
Another solution similar to vue.js with the vue-loader to pre-process the .vue files in the deployment step.

**Comment by stefanhoelzl** [comment]
I added some missing documentation how to specify external templates in `vuepy.yml`.

These allow you to write your html templates in a separate `.html`-file.
In addition with specifying your `.css`-files also in `vuepy.yml` you do not need to write any html or css in python strings.

It is still in separate files and not in one single file, but it may be a workaround for the missing .vue file support for now.
